content/en/events/2024/kcseu/social.md

```diff
@@ -21,4 +32,4 @@ them.
 
 **NOTE:** Guests must adhere to all KubeCon health & safety requirements.
 
-More information on Guest registration to come, closer to the event.
+To bring a guest, please email community@kubernetes.io with their information.
```

Comment:
https://github.com/kubernetes/contributor-site/pull/479#issuecomment-1969673213 
Should this be summit-staff@kubernetes.io instead?

Reply:
@nitishfy It's in fact `summit-team@kubernetes.io` ?

Reply:
Resolved